### DISCUSSION TITLE
`puppeteer-core`

### DISCUSSION THREAD

**Discussion by mbardelmeijer** [spatie/browsershot#905] [General]
Is it possible to use `browser shot` with `puppeteer-core,` where a singular browser installation is installed server-wide, without installation in the `~/.cache/puppeteer` folder? 

From what I can see, `puppeteer-core` is a drop-in replacement of `puppeteer`, but without the downloading of the Chrome browser. 

Goal is to have a singular installation of Chrome across the webhost, where multiple unix users can share that singular installation to safe disk space. With 100+ Chrome installations of a few GiB each, disk space grows :)

Anyone had any success with this approach?

**Comment by frtess** [comment]
Wondering the same thing. Did you figure it out ?

**Reply by mbardelmeijer** [reply]
Not yet; still awaiting input to see if anyone found a way to deploy this
